### PR TITLE
[cached@5.0.2] first definition types

### PR DESCRIPTION
**Pull request by Juraj-Masiar**
Library:
https://github.com/groupon/node-cached
`npm install --save cached`

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/cached/tslint.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        "array-type": false,
+        "ban-types": false,
+        "comment-format": false,
+        "dt-header": false,
+        "no-padding": false,
+        "no-unnecessary-generics": false,
+        "object-literal-shorthand": false,
+        "prefer-template": false,
+        "semicolon": false,
+        "unified-signatures": false
```
Comment (andrewbranch) [inline types/cached/tslint.json:13]: Can we delete these please?

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/cached/tslint.json (hunk 1 above)
Comment (Juraj-Masiar) [reply types/cached/tslint.json]: sure thing!

**Review by andrewbranch — APPROVED** [review]
Thanks!

**Comment by Juraj-Masiar** [comment]
Ready to merge

**Pull request merged**